File: Persona2.py
# coding=utf-8
class Persona:
    nombre = ""
    edad = 0
    email = ""

#aquí defino un constructor, __init__ es un método predefinido de Python no lo debes de tratar de modificar
    def __init__(self, nombre, edad, email):
        self.edad = edad
        self.nombre = nombre
        self.email = email
#aquí defino un método
    def mostrarCredencial(self):
        print "Tu nombre es: ", self.nombre
        print "Tu edad es: ", self.edad
        print "Tu email es: ", self.email
#aquí defino al objeto y a su instancia
newPersona = Persona("Alfredo", 28, "ejemplo@hotmail.com")
newPersona.mostrarCredencial()

File: Persona2.php
<?php


class Persona2
{
    public $nombre;
    public $edad;
    public $email;

    public function setPersona($nombre, $edad, $email)
    {
        $this->nombre = $nombre;
        $this->edad = $edad;
        $this->email = $email;
    }

    public function mostrarCredencial()
    {
        echo "Tu nombre es: ".$this->nombre;
        echo "Tu edad es: ".$this->edad;
        echo "Tu email es: ".$this->email;
    }
}

$newPersona = new Persona2();
$newPersona->setPersona("Alfredo", 28, "ejemplo@mail.com");
$newPersona->mostrarCredencial();
